bedrock/newsletter/views.py

```diff
@@ -53,12 +52,12 @@ def set_country(request, token):
     if countrycode:
         initial["country"] = countrycode.lower()
 
-    form = CountrySelectForm("en-US", data=request.POST or None, initial=initial)
+    form = CountrySelectForm("en-US", None, initial=initial)
```

Comment:
You can drop the 2nd arg `None` here if you'd like since it's the default. The locale is our own addition we defined in the `CountrySelectForm` class, the rest is just standard Django form args.